packages/utilities/psammead-test-helpers/index.test.jsx

```diff
@@ -234,13 +234,12 @@ describe('Psammead test helpers', () => {
     <HelmetWithContent />,
   );
 
-  it('should create a snapshot from an async it() block with component and done props', async done => {
-    const data = Promise.resolve('Foobar');
-    const component = (
+  it('should create a snapshot from an async it() block', async () => {
+    const data = await Promise.resolve('Foobar');
+    await testHelpers.matchSnapshotAsync(
       <div>
-        <p>{await data}</p>
-      </div>
+        <p>{data}</p>
+      </div>,
```

Comment:
what is the comma used here for?

Reply:
Prettier is taking `<div><p>{data}</p></div>` as the first param in the function call, and because it's multi-line prettier adds a comment for future params to be passed. 

Alternatively it could be:
```
const component = (
  <div>
    <p>{data}</
  </div>
);
await testHelpers.matchSnapshotAsync(component);
```

Reply:
Also I agree the comma currently looks gross and confusing :trollface:

Reply:
happy with how it is, just wanted to check. Thought it might be a linting thing